.travis.yml

```diff
@@ -30,6 +30,7 @@ cache:
   directories:
   - $HOME/.m2
 script:
+  - mvn clean package -q -Dmaven.javadoc.skip=true -Dhadoop=3.2 -Dspark=2.4 -Dscala=2.12
```

Comment:
Another way to add jdk11 testing in maven/travis is to add a pom tag with <java.version>=1.11? And then explicitly enable this tag in this script.

Reply:
Sounds fair enough.